package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@curi-bio/mantarray-frontend-components",
-  "version": "0.6.0",
+  "version": "0.3.0",
```

Comment:
Last release was `0.2.0`, so we can still mark this version as `0.3.0`

Reply:
Got it, that's my bad, I thought every big, unbroken merge required an increase in the minor version